modules/nodes-cluster-enabling-features-cluster.adoc

```diff
@@ -1,48 +1,40 @@
 // Module included in the following assemblies:
 //
 // * nodes/nodes-cluster-enabling-features.adoc
-// * post_installation_configuration/cluster-tasks.adoc
 
 [id="nodes-cluster-enabling-features-cluster_{context}"]
-= Enabling Technology Preview features using feature gates
+= Enabling feature sets using the web console
 
-You can turn on Technology Preview features on for all nodes in the cluster by
-editing the Feature Gate Custom Resource, named `cluster`, in the `openshift-config` project.
-
-The following Technology Preview features are enabled by feature gates:
-
-* `RotateKubeletServerCertificate`
-
-* `SupportPodPidsLimit`
-
-[IMPORTANT]
-====
-Turning on Technology Preview features using the Feature Gate Custom Resource cannot be undone and prevents upgrades.
-====
+You can use the {product-title} web console to enable feature sets for all nodes in the cluster by editing the Feature Gate custom resource (CR).
 
 .Procedure
 
-To turn on the Technology Preview features for the entire cluster:
+To enable feature sets:
 
 . In the {product-title} web console, switch to the *Administration* -> *Custom Resource Definitions* page.
 
 . On the *Custom Resource Definitions* page, click *FeatureGate*.
 
-. On the *Custom Resource Definitions* page, click *Actions* -> *View Instances*.
+. On the *featuregates.config.openshift.io* page, click the *Instances* tab.
 
-. On the *Feature Gates* page, click *Create Feature Gates*.
+. Click the *cluster* instance then click the *YAML* tab.
 
-. Add the `featureSet` parameter:
+. Edit the *cluster* instance to add the `featureSet` parameter and specific feature sets:
```

Comment:
there is no easier way than this to reach a config CR? @jhadvig can you confirm?